pkg/util/helper/policy.go

```diff
@@ -33,6 +35,33 @@ func SetDefaultSpreadConstraints(spreadConstraints []policyv1alpha1.SpreadConstr
 	}
 }
 
+// SetDefaultClusterTolerations set default notReadyToleration and unreachableToleration if not set.
+func SetDefaultClusterTolerations(placement *policyv1alpha1.Placement, notReadyToleration, unreachableToleration corev1.Toleration) {
```

Comment:
This function is a little redundant. We can ignore which `corev1.Toleration` is set, suggest:

```
func SetDefaultClusterTolerations(placement *policyv1alpha1.Placement, tolerations ...corev1.Toleration)
```
or
```
func SetDefaultClusterToleration(placement *policyv1alpha1.Placement, toleration corev1.Toleration)
```